src/cljs/nr/gameboard/player-stats.cljs

```diff
@@ -43,18 +43,21 @@
          (ctrl :memory [:div label (when (neg? unused) [:div.warning "!"])]))))))
 
 (defn- display-special-memory
-  [memory]
+  ([memory] (display-special-memory memory false))
+  ([memory icon?]
   (when-let [only-for (->> (:only-for memory)
                            (filter #(pos? (:available (second %))))
                            (into {})
                            not-empty)]
     [:div
      (str "("
           (join "), (" (for [[mu-type {:keys [available used]}] only-for
-                             :let [unused (max 0 (- available used))]]
-                         (str unused " of " available
-                              " " (capitalize (name mu-type))
-                              " [mu] unused")))
+                             :let [
+                               unused (max 0 (- available used))
+                               mu-type-name (capitalize (name mu-type))
+                              ]]
+                         (if icon? [:<> unused "/" available " " mu-type-name " " [:span.anr-icon.mu]]
+                             (tr [:game.special-mu-count] unused available mu-type-name))))
           ")")]))
```

Comment:
You're missing a parenthesis at the end of `display-special-memory` since you added the second signature. Won't build without it.

This needs some more work since it doesn't display properly after it builds. I think the issue is that this is wrapped entirely in a `(str)` function.   
![image](https://user-images.githubusercontent.com/5953664/133688829-ce112525-23fc-47d8-ac41-cb11f7fa3096.png)